ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/ChainSel.hs

```diff
@@ -745,17 +743,23 @@ chainSelectionForBlock cdb@CDB{..} blockCache hdr punish = do
               -- Clear the tentative header
               prevTentativeHeader <- swapTVar varTentativeHeader SNothing
 
-              -- Update the followers
-              --
-              -- 'Follower.switchFork' needs to know the intersection point
-              -- (@ipoint@) between the old and the current chain.
-              let ipoint = castPoint $ Diff.getAnchorPoint chainDiff
-              followerHandles <- Map.elems <$> readTVar cdbFollowers
-              forM_ followerHandles $ \followerHandle ->
-                fhSwitchFork followerHandle ipoint newChain
+              case chainSwitchType of
+                AddingBlock      -> pure ()
```

Comment:
Please add a comment here about why it's safe/important to skip `fhSwitchFork` here.

And update the PR description.

(As we just agreed on the call; this is merely reifying that.)